sound/soc/sof/ipc4-topology.c

```diff
@@ -1957,6 +1957,46 @@ static void sof_ipc4_put_queue_id(struct snd_sof_widget *swidget, int queue_id,
 	ida_free(queue_ida, queue_id);
 }
 
+static int sof_ipc4_set_copier_sink_format(struct snd_sof_dev *sdev,
+					   struct snd_sof_widget *swidget, int sink_id)
```

Comment:
may I ask why you call it sink_id when you're passing the source_queue_id?

Reply:
> may I ask why you call it sink_id when you're passing the source_queue_id?

The sink_id here is the source widget's sink_id which is equal to sroute->src_queue_id.